Tutorials/Supervised/ED.py

```diff
@@ -0,0 +1,267 @@
+import json
```

Comment:
We could also generate the data directly using the ED solver in NetKet, but that's a matter of taste, no problem with this